src/input/PdbInputHandler.ts

```diff
@@ -0,0 +1,21 @@
+import { DebuggingCommand, DebuggingInputHandler } from "./DebuggingInputHandler";
+
+/**
+ * Implementation of a DebuggingInputHandler for Python with Pdb
+ */
+export class PdbInputHandler extends DebuggingInputHandler {
+    protected override buildCommandMap(): Map<DebuggingCommand, string> {
+        return new Map([
+            [DebuggingCommand.StepInto, "s"],
+            [DebuggingCommand.StepOver, "n"],
+            [DebuggingCommand.Continue, "c"]
```

Comment:
I just want to point out that like with breakpoints, you can call methods like https://docs.python.org/3/library/bdb.html#bdb.Bdb.set_step directly instead of working with pdb's input and output. But I can see how this has advantages.

Reply:
Is this really possible after using set_trace? The code will be run until completion and blocks at breakpoints. This blocking is done because of the waiting for input. I can't call other methods on the debugger because the entire Python thread is halting. Autocompletion for instance no longer works while debugging. This blocking is also desired as it allows the user to have the time to pick their next action while interrupting the code run.  I'm not sure if there is an entrypoint to feed Pdb a chunk of code and then calling the corresponding methods on the debugger. I would prefer this approach, but I don't see how at this moment.

Reply:
You're right, I think it would require using the base bdb more directly and rebuilding much of what pdb does, which doesn't seem worth it. Maybe you could override `Pdb.cmdloop` which is where it reads from stdin and do your own callbacks and waiting? But I don't see any issues with your current approach at the moment.

> Autocompletion for instance no longer works while debugging

That's a very interesting point. You could probably set up a second worker dedicated to autocomplete, but that's also probably not worth it.